lib/utils/class-context.php

```diff
@@ -22,6 +22,11 @@ public static function is_maintenance_mode() {
 			&& true === constant( 'WPCOM_VIP_SITE_MAINTENANCE_MODE' );
 	}
 
+	public static function is_overdue_locked() {
+		return defined( 'VIP_OVERDUE_LOCKOUT' )
+			&& true === constant( 'VIP_OVERDUE_LOCKOUT' );
```

Comment:
For simplicity, we can skip the `constant()` function here, as it's only really needed when the constant name is not known beforehand (a variable).
```suggestion
			&& true === VIP_OVERDUE_LOCKOUT;
```

Reply:
@nickdaugherty, we prefer `constant()` these days because it's better testable.